configuration/pih/scripts/global/zl.js

```diff
@@ -88,3 +88,62 @@ function setUpExpandableContacts(badPhoneNumberMsg) {
 
 }
 
+function datepickerAsyncValidation(badDateInTheFutureMsg,badStartDateGreaterThanEndDateMsg){
```

Comment:
Looks good, thanks @louidorjp !

One thought: could we rename this function to better describe what it does?  Maybe something datepickerStartAndEndDateValidation? (I'm not in love with that one either, maybe @brandones  has a better idea)

I also think it would make sense to take in the start and end date elements as parameters so that we we could use this on multiple forms that might have 1 to many dates need to be vallidated, so, for example, you'd set up the validation in this form like this:

```
datepickerStartAndEndDateValidation(jq('#startDate-0), jq('#endDate-0'), ('<uimessage code="pihcore.date.error.inTheFuture"/>','<uimessage code="pihcore.startDate.greather"/>');

datepickerStartAndEndDateValidation(jq('#startDate-1), jq('#endDate-1'), ('<uimessage code="pihcore.date.error.inTheFuture"/>','<uimessage code="pihcore.startDate.greather"/>');
```

Reply:
@mogoodrich it's a good idea. I changed the name of the method